src/types/runtime/gravity/Feature.ts

```diff
@@ -14,6 +14,8 @@ export const Feature = Record({
   id: String,
   name: String,
   description: String.Or(Null),
+  subheadline: String.Or(Null),
+  callout: String.Or(Null),
```

Comment:
So one thing to note here is — nothing in production is using the runtime validation yet since I only enabled it on the stuff we're using for the new feature pages — but if this was deployed before these fields were in the Gravity responses it'd error since it'd expect the keys to at least be defined. I'm going to write a lil wrapper for the checks that ignores the error and just logs it for the time being.